spec/shopify_transporter/shopify/order_spec.rb

```diff
@@ -104,6 +118,7 @@ def top_level_attributes_row(hash)
           *Array.new(line_item_attributes.size, nil),
           *Array.new(tax_line_attributes.size, nil),
           *Array.new(metafield_attributes.size, nil),
+          *Array.new(transaction_attributes.size, nil),
```

Comment:
minor: Maybe move this up a line so it's before `metafield_attributes`. However, it doesn't change the expected output here.